components/yaml/introduction.rst

```diff
@@ -250,6 +233,74 @@ representation of the object.
     parsers will likely not recognize the ``php/object`` tag and non-PHP
     implementations certainly won't - use with discretion!
 
+Handling Invalid Types
+......................
+
+By default the parser will encode invalid types as "null". You can make the
+parser throw exceptions by using the ``PARSE_EXCEPTION_ON_INVALID_TYPE`` flag:
+
+    $yaml = '!php/object:O:8:"stdClass":1:{s:5:"foo";s:7:"bar";}';
+    Yaml::parse($yaml, Yaml::PARSE_EXCEPTION_ON_INVALID_TYPE); // throws an exception
+
+Inversely you can use ``DUMP_EXCEPTION_ON_INVALID_TYPE`` when dumping:
```

Comment:
`Inversely` ->  `Similarly` ? Both flags are identical and work the same way. "Inversely" implies that they are different ... but only the underlying process is different (parsing or dumping).